.gitignore

```diff
@@ -5,4 +5,12 @@ coverage.xml
 __pycache__
 .pytest_cache
 db.sqlite3
-data
\ No newline at end of file
+data
+.vscode
+
+# Need to use real malware to test antivirus, appending .malz to any potentially malicious file
+# and placing it in the gitignore so it doesn't accidentally get pushed
+*.malz
```

Comment:
Sorry for the late push, just commenting on these for future reference!
This extension was put in place to dearm any malware samples I had on my own system (although it was more habit because extensions don't really matter to Linux) to generate antivirus logs more interesting than just 10 eicar files. This has since been removed and instructions have been included for anyone who would like to test using live malware.